CHANGELOG.md

```diff
@@ -1669,3 +1670,4 @@
 [@eagletmt]: https://github.com/eagletmt
 [@apiology]: https://github.com/apiology
 [@alexdowad]: https://github.com/alexdowad
+[@alexdowad]: https://github.com/alexdowad
```

Comment:
:) That came from merging 2 commits which were both originally based on the same `master`. Removing now.